src/ucp/wireup/wireup_cm.c

```diff
@@ -185,17 +183,14 @@ static ssize_t ucp_cm_client_priv_pack_cb(void *arg,
 
         tl_bitmap |= UCS_BIT(rsc_idx);
         if (ucp_ep_config(tmp_ep)->p2p_lanes & UCS_BIT(lane_idx)) {
-            tl_ep_params.field_mask = UCT_EP_PARAM_FIELD_IFACE |
-                                      UCT_EP_PARAM_FIELD_PATH_INDEX;
-            tl_ep_params.iface      = ucp_worker_iface(worker, rsc_idx)->iface;
-            tl_ep_params.path_index = ucp_ep_get_path_index(tmp_ep, lane_idx);
-            status = uct_ep_create(&tl_ep_params, &tl_ep);
+            status = ucp_wireup_ep_connect(tmp_ep->uct_eps[lane_idx], 0,
+                                           rsc_idx,
+                                           ucp_ep_get_path_index(tmp_ep,
+                                                                 lane_idx),
+                                           0, NULL);
             if (status != UCS_OK) {
-                /* coverity[leaked_storage] */
                 goto out;
             }
-
-            ucp_wireup_ep_set_next_ep(tmp_ep->uct_eps[lane_idx], tl_ep);
```

Comment:
please check that not using `ucp_wireup_ep_set_next_ep` does not produce any error cases
(`ucp_wireup_ep_connect` uses `ucp_proxy_ep_set_uct_ep`)

Reply:
yes, it should be `ucp_proxy_ep_set_uct_ep`, since `LOCAL_CONNECTED` flag should be set when `uct_ep_connect_to_ep()` has be called - added assert to check that `LOCAL_CONNECTED` isn't set when calling `uct_ep_connect_to_ep()`